src/com/session/GoogleResp.java

```diff
@@ -17,45 +20,58 @@
 	private static final long serialVersionUID = 1L;
 
 	public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-		
-		String accessToken=null;
+
 		String authcode = req.getParameter("code");
-		String email=null;
-		boolean isPresent=false;
-		System.out.println(authcode);
+		String error = req.getParameter("error");
+		GoogleInfo post = null;
+
+		if ("access_denied".equals(error)) {
+			resp.sendRedirect("login?access_denied");
```

Comment:
after this, their is no return statement, so code will still continue to execute